grpc/src/main/java/com/replica/RouterImpl.java

```diff
@@ -327,8 +327,16 @@ private void augmentLegsForPt(ResponsePath path) {
                 path.getLegs().add(RouterConverters.toCustomWalkLeg(thisLeg, travelSegmentType));
             } else if (leg instanceof Trip.PtLeg) {
                 Trip.PtLeg thisLeg = (Trip.PtLeg) leg;
-                path.getLegs().add(
-                        RouterConverters.toCustomPtLeg(thisLeg, gtfsFeedIdMapping, gtfsLinkMappings, gtfsRouteInfo));
+
+                long startTime = System.currentTimeMillis();
+                CustomPtLeg customPtLeg = RouterConverters.toCustomPtLeg(thisLeg, gtfsFeedIdMapping, gtfsLinkMappings, gtfsRouteInfo);
+                double durationSeconds = (System.currentTimeMillis() - startTime) / 1000.0;
+                if (statsDClient != null) {
+                    statsDClient.histogram("routers.to_custom_pt_leg_seconds", durationSeconds);
```

Comment:
This looks great, but could we add the basic tags (mainly release name and region) here to make it easier to search/filter with this metric? They're stored in `customTags` in this class, but need to be converted from map -> strings to pass into `histogram`. I think something like this would do the trick:
```
MetricUtils.applyCustomTags(new String[0], customTags)
```

Reply:
good catch, i missed that we apply custom tags to all metrics! pushed a fix